### PR TITLE
Document how to connect with a jupyterhub

### PR DESCRIPTION
**Pull request by yuvipanda**
I've only written the outline. It needs to be fleshed out, edited and tested.

Fixes https://github.com/2i2c-org/infrastructure/issues/3691

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,9 +175,190 @@ The documentation should help configure the BinderHub service to:
...
+   ```
+
+2. We want the binderhub to be available under `http://{{hub url}}/services/binder`, because
+   that is what `jupyterhub-fancy-profiles` expects. Eventually we would also want authentication
+   to work correctly. For that, we must set up binderhub as a [JupyterHub Service](https://jupyterhub.readthedocs.io/en/stable/reference/services.html).
+   This provides two things:
+
+   a. Routing from `{{hub url }}/services/{{ service name }}` to the service, allowing us to
+   expose the service to the external world using JupyterHub's ingress / loadbalancer, without
+   needint a dedicated ingress / loadbalancer for BinderHub.
```
Comment (sgibson91) [inline README.md:199]:
```suggestion
   needing a dedicated ingress / loadbalancer for BinderHub.
```

File: README.md [hunk 2]
```diff
@@ -176,9 +175,190 @@ The documentation should help configure the BinderHub service to:
...
+
+4. Access the JupyterHub itself, using the external IP you got from step 5 of the section
+   above (not `{{ hub IP }}/services/binder/`). Once you log in (you can use _any_ username
+   and password), you should see a UI that allows you to choose two pre-existing
+   images (pangeo and scipy), specify your own image, or 'build' your own image.
+   The last option lets you access the binder functionality! Test it out :)
+
+From now on, you can customize this JupyterHub as you would any other JupyterHub set up
+using z2jh. The [customization guide](https://z2jh.jupyter.org/en/stable/jupyterhub/customization.html)
+contains many helpful. In particular, you probably want to set up more restrictive
```
Comment (sgibson91) [inline README.md:359]:
```suggestion
contents may be helpful. In particular, you probably want to set up more restrictive
```

I think this is what this sentence is supposed to say?

Comment (sgibson91) [reply README.md:359]: Or perhaps "contains many helpful examples"?

**Review by sgibson91 — APPROVED** [review]
LGTM!

File: README.md (hunk 2 above)
Comment (yuvipanda) [reply README.md]: Ah, looks like my brain stopped mid sentence! "contains many helpful examples" is close, I pushed a change with something like that.

**Pull request merged**